### ISSUE TITLE
how can we calculate the true positive and false positive , accuracy, precision 

### ISSUE THREAD

**Issue by dheeraj1129** [saimj7/People-Counting-in-Real-Time#9]
#accuracy #tpr #fpr #presicion

**Comment by saimj7** [comment]
You need to run the predictions on test data and then compare them with actual test labels, which would greatly involve refactoring the code. One simple example is to use `classification_report` from sklearn.

**Comment by dheeraj1129** [comment]
sir, I want to train a mobile net but I don't have a dataset. I just want you to send me all necessary things which needed for train the model for people detection. I want to train a mobile net on my laptop for further work. please I am very beginner in ML so please help me out. I need to submit my project in college I run your project but they told me to train mobile net so please help me out and send me all the necessary things like dataset and program. i am waiting for your response.

**Comment by saimj7** [comment]
This book explains training SSD and other object detectors from scratch: [https://www.pyimagesearch.com/deep-learning-computer-vision-python-book/], you could contact them or give it a try.

**Comment by saimj7** [comment]
This book explains training SSD and other detectors from scratch: [](https://www.pyimagesearch.com/deep-learning-computer-vision-python-book/), you could contact them or give it a try.
